book/src/prerequisites.md

```diff
@@ -1,21 +1,76 @@
-# Installation
+# Prerequisites
+
+There are a few recommended ways to set up your workstation in order to develop `gtk-rs` applications.
+Let us go through them one by one.
+
+## Cargo
+
+[Cargo](https://doc.rust-lang.org/cargo/index.html) is Rust's build system and package manager.
+If following the book is all you care about, using only Cargo will work fine for you.
 
 Let us begin by installing all necessary tools.
-First follow the instructions on the [GTK website](https://www.gtk.org/docs/installations/) to install GTK 4.
+First follow the instructions on the [GTK website](https://www.gtk.org/docs/installations/) in order to install GTK 4.
 Then install Rust with [rustup](https://rustup.rs/).
 
 Now create a new project by executing:
 ```bash
 cargo new my-gtk-app
 ```
 
-Add the following lines to your `Cargo.toml` and you are ready to go!
+Add the following lines to your `Cargo.toml`.
 ```toml
 [dependencies.gtk]
 version = "0.1"
 package = "gtk4"
 ```
 
-# Buildsystem
+Now you can run your application by executing:
+```bash
+cargo run
+```
+
+## Cargo + Meson
+
+Cargo is *nearly* enough, but it is not well suited for handling resources such as icons or UI files.
+That is why we recommend to use [Meson](https://mesonbuild.com/) on top of it.
+It is cross-platform and its syntax is very readable.
+Meson takes care of
+- translations
+- building and installing GResources
```

Comment:
Could use a link to what a GResource is https://developer.gnome.org/gio/stable/GResource.html

Reply:
I think, I'll link to the corresponding chapter in the book instead.
Good point though :)